docs/content/how-to/ci/ai-assisted-ci-jobs.md

```
@@ -171,43 +167,15 @@ When addressing feedback, the bot follows these rules:
-            J -->|Yes| G
-            J -->|No| F
-        end
-    end
+For a single PR, trigger the review agent with:
 
-    subgraph "External Systems"
-        D <--> GH[(GitHub API<br/>github.com)]
-        L <--> CLAUDE[Claude API<br/>via Vertex AI]
-        M <--> FORK[(GitHub Fork<br/>hypershift-community)]
-    end
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)